ReactiveObjC/RACSignal.h

```diff
@@ -180,7 +180,7 @@ NS_ASSUME_NONNULL_BEGIN
 /// Additional methods to assist with unit testing.
 ///
 /// **These methods should never ship in production code.**
-@interface RACSignal (Testing)
+@interface RACSignal<__covariant ValueType> (Testing)
```

Comment:
No, this change was not enforced by the compiler—I just added it so that `asynchronousFirstOrDefault` could have access to `ValueType`. Note that `RACSignal (Debugging)` does not have it as part of this change.